src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
All fine with me!  I'll give it another look tomorrow. Please also note the changed behaviour of _push and _concat, they now modify the original arrays instead of creating a new one, should probably have mentioned that in the pull request. Also, I have no access to GM for the next week due to being on a trip with linux laptop only. I can ask a mate of mine to run the test, or maybe you could create a branch for me to work on, and maybe run the tests tomorrow somewhere? I could write some tests for new behaviour as well.